common/src/commonMain/kotlin/com/mirego/nwad/domain/CreateMomentUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.mirego.nwad.domain
+
+import com.mirego.trikot.streams.reactive.promise.Promise
+
+interface CreateMomentUseCase {
+    fun createNewMoment(createMomentData: CreateMomentData): Promise<Unit>
+}
+
+data class CreateMomentData(val jpegImageContent: ByteArray, val title: String, val parentMoment: String? = null)
```

Comment:
ParentMoment, je suis pas certain du nom. `parentMomentId`?

Reply:
Lorsque l'on va créer un moment a partir d'un autre moment. C'est subject to change.